src/modules/sub-items/sub.items.module.js

```diff
@@ -969,6 +962,8 @@ export default class SubItemsModule extends Component {
 
         const selectedPageLocationsIds = this.getPageSelectedLocationsIds();
 
+        this.updateTrashModal();
```

Comment:
Not sure it this should be fired every time when React is rerendering the list view